plugins/generator-1.19.4/forge-1.19.4/procedures/team_add_entity.java.ftl

```diff
@@ -0,0 +1,6 @@
+{
+	Level _level = ${input$entity}.level();

```

Comment:
>  just save entity into tmp variable first so procedure providing entity is not called multiple times. Applies to other cases like this too.

Right now, entity is obtained twice, but could be just once into tmp variable. Matters eg. for in range procedure which is resource heavy

Also check 1.19.4, it has no .level() yet, only .level